src/bridge-canister/src/memory.rs

```diff
@@ -8,7 +8,7 @@ pub const OPERATIONS_MEMORY_ID: MemoryId = MemoryId::new(3);
 pub const OPERATIONS_LOG_MEMORY_ID: MemoryId = MemoryId::new(4);
 pub const OPERATIONS_MAP_MEMORY_ID: MemoryId = MemoryId::new(5);
 pub const PENDING_TASKS_MEMORY_ID: MemoryId = MemoryId::new(6);
-pub const LOG_SETTINGS_MEMORY_ID: MemoryId = MemoryId::new(7);
+pub const LOG_SETTINGS_MEMORY_ID: MemoryId = MemoryId::new(77);
```

Comment:
is this a typo?

Reply:
That is debugging forgotten to be removed. Fixed it.